docs/runbooks/KubeMacPoolDuplicateMacsFound.md

```diff
@@ -1,32 +1,40 @@
 # KubeMacPoolDuplicateMacsFound
+<!--apinnick, Oct. 2022-->
 
 ## Meaning
 
-KubeMacPool is responsible to allocate MAC Addresses and prevent MAC Address conflicts.  
-Upon starting KubeMacPool, it scans the cluster for MAC Addresses in VMs set on the managed namespaces.  
-In case it finds duplicates it will raise this alert.
+This alert fires when `KubeMacPool` detects duplicate MAC addresses.
+
+`KubeMacPool` is responsible for allocating MAC addresses and preventing MAC address conflicts. When `KubeMacPool` starts, it scans the cluster for the MAC addresses of virtual machines (VMs) in managed namespaces.
 
 ## Impact
 
-Duplicate MAC Addresses on the same LAN may cause unexpected and various connectability issues in the network.
+Duplicate MAC addresses on the same VLAN might cause network issues.
 
 ## Diagnosis
 
-- Find the `kubemacpool-mac-controller` namespace and pod  
-  `kubectl get pod -A -l control-plane=mac-controller-manager --no-headers -o custom-columns=":metadata.namespace,:metadata.name"`
-- Extract the confliciting Mac Addresses from the `kubemacpool-mac-controller` pod  
-  `kubectl logs -n <kubemacpool-namespace> <kubemacpool-mac-controller-pod-name> | grep "already allocated"`  
-  Those lines have all the information that is needed:
-    - The Mac Address which has a conflict.
-    - The current entry in the data base (VM namespace, name and interface).
-    - The entry that rejected due to conflict (VM namespace, name and interface).  
+1. Obtain the namespace and the name of the `kubemacpool-mac-controller` pod:
+  ```bash
+  $ kubectl get pod -A -l control-plane=mac-controller-manager --no-headers \
+    -o custom-columns=":metadata.namespace,:metadata.name"
+  ```
+
+1. Obtain the duplicate MAC addresses from the `kubemacpool-mac-controller` logs:
```

Comment:
```suggestion
2. Obtain the duplicate MAC addresses from the `kubemacpool-mac-controller` logs:
```

Reply:
Numbering for markdown can be "1." for each step. It will resolve correctly.

Reply:
I don't follow..
you want it to be lie this?
```
1. Obtain the namespace and the name of the kubemacpool-mac-controller pod:
...
1. Obtain the duplicate MAC addresses from the kubemacpool-mac-controller logs:
```

if not - please fix. If you do - then why? these are consecutive items in the same list, why would they have the same number?

Reply:
Normally markdown does not require different number for ordered lists.
![image](https://user-images.githubusercontent.com/31725355/198882448-d6306160-0ac9-4109-ae7e-fcd90186ea16.png)

If you merge this and the numbers are not correct, I will fix them.